### PR TITLE
Update ja_JP translation

### PR DESCRIPTION
**Pull request by YTJVDCM**
I updated and translated new strings.

### CODE REVIEW

File: src/i18n/ja_JP.json [hunk 1]
```diff
@@ -487,5 +591,44 @@
...
+  "oobe.general.subtitle": "",
+  "oobe.general.text": "",
+  "oobe.audio.title": "オーディオ",
+  "oobe.audio.subtitle": "",
+  "oobe.audio.text": "Ciderにはリッチで高品質なオーディオを提供するための、カスタマイズ、デザインされたオーディオスタック機能があります。\nCider Adrenaline、Atmosphere Realizer、ならびにSpatailized Audioがあります。\nこれらの機能を有効化するには、\"先進的な機能\"を有効化する必要があります。\n先進的な機能を有効化すると、これらの強化機能をアプリ設定のCider Audio Labsから利用することができるようになります。",
+  "oobe.audio.advancedFunctionality": "",
+  "oobe.visual.title": "ビジュアル",
+  "oobe.visual.subtitle": "",
+  "oobe.visual.text": "",
+  "oobe.visual.layout.text": "Ciderには2つの異なるウィンドウレイアウトがあります。\nMarverickはウィンドウ上部にプレイヤーのある、iTunes風のレイアウトです。\nMojaveはCider Collectiveによって作成された新たな試作品です。\n\nこれらのレイアウトはいつでも設定から変更することができます。",
```
Comment (maikirakiwi) [inline src/i18n/ja_JP.json:620]: Marverick-> Maverick

File: src/i18n/ja_JP.json [hunk 2]
```diff
@@ -309,14 +389,18 @@
...
+  "settings.header.audio.quality.high.description": "256 kbps",
   "settings.header.audio.quality.standard": "高効率",
+  "settings.header.audio.quality.standard.description": "64 kbps",
   "settings.option.audio.seamlessTransition": "曲間なしで再生",
   "settings.option.audio.enableAdvancedFunctionality": "先進的な機能",
   "settings.option.audio.enableAdvancedFunctionality.description": "AudioContextを有効にすると、オーディオノーマライズ、空間オーディオ、イコライザーなどの機能を使用できますが、音が途切れるかもしれません。",
   "settings.warn.audio.enableAdvancedFunctionality.lowcores": "あなたのPCがこの処理に耐えられないかもしれません。",
   "settings.option.audio.audioLab": "Cider Audio Lab",
-  "settings.option.audio.audioLab.description": "An assortment of in-house developed audio effects for Cider.",
+  "settings.option.audio.audioLab.description": "Ciderのために開発された室内音響設定の数々",
```
Comment (maikirakiwi) [inline src/i18n/ja_JP.json:400]: ちょっと意味が微妙に違う。
「in-house developed」って室内の意味ではなく、More like 自社開発?

I think.

**Review by maikirakiwi — CHANGES_REQUESTED** [review]

**Comment by YTJVDCM** [comment]
Thanks for your request.
I misunderstood the meaning of the word.
And I fixed it.

**Comment by maikirakiwi** [comment]
迅速に対応いただきありがとうございます。
I will be merging this now.

**Pull request merged**